docs/docs/100-reference/01-command-line/acorn_run.md

```diff
@@ -73,39 +83,23 @@ acorn run [flags] IMAGE|DIRECTORY [acorn args]
   acorn run --notify-upgrade myorg/hello-world:v#.#.# myapp
   # To proceed with an upgrade you've been notified of:
   acorn update --confirm-upgrade myapp
-
 ```
 
 ### Options
 
 ```
-      --annotation strings        Add annotations to the app and the resources it creates (format [type:][name:]key=value) (ex k=v, containers:k=v)
```

Comment:
Would still like to see all the available options on the docs site. If I was searching for an option I want to use on the docs site, I'd like it to pop up here.

Reply:
since these are autogenerated it becomes hard also the main goal of this is to get users away from using the run flags and instead pushing profiles so unsure if we want this there

Reply:
ahh